core/src/test/scala/com/yahoo/maha/core/query/oracle/OracleQueryGeneratorTest.scala

```diff
@@ -3111,4 +3111,291 @@ class OracleQueryGeneratorTest extends BaseOracleQueryGeneratorTest {
        """.stripMargin
     result should equal (expected) (after being whiteSpaceNormalised)
   }
+
+  test("Outer Group by Tests") {
```

Comment:
Can we make test names more verbose to what they are doing ... e.g. successfully generate valid outer group by query ...